src/views/infra/InfraSummary.vue

```diff
@@ -41,11 +41,102 @@
         </a-button>
         <a-modal
           :title="$t('SSL Certificate')"
-          v-model="sslFormVisible"
-          @ok="handle">
-          <p>Some contents...</p>
-          <p>Some contents...</p>
-          <p>Some contents...</p>
+          :visible="sslFormVisible"
+          :footer="null"
```

Comment:
Does not declaring `footer` cause to show any footer?

Reply:
I'm not too sure, I'll have a play with it and if not declaring this prop has the same effect then I will remove it. 👍

Reply:
I've had a look through the docs and tested it, and it appears that you need to specify the footer as null, or it will display by default.